src/components/videoCard.jsx

```diff
@@ -42,27 +42,45 @@ export function VideoCard({ video }) {
     }
   };
 
-  const addToLikesHandler = () =>
+  const addToLikes = () =>
     isLiked
       ? resourcesApiDispatch({ type: REMOVE_FROM_LIKES, payload: id })
```

Comment:
`resourcesApiDispatch` can be better named

Reply:
Thanks for the suggestion, I have fixed it in the latest commit